java-squid/src/main/java/org/sonar/java/se/ProgramState.java

```diff
@@ -106,7 +106,6 @@ private ProgramState(ProgramState ps, PMap<SymbolicValue, Object> newConstraints
     constraintSize = ps.constraintSize +1;
     visitedPoints = ps.visitedPoints;
     this.stack = ps.stack;
-    symbolicValueRelations = ps.symbolicValueRelations;
   }
 
   private ProgramState(ProgramState ps, List<SymbolicValueRelation> newRelation) {
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) Private constructor 'org.sonar.java.se.ProgramState(ProgramState,List)' is never used. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://dory.sonarsource.com/coding_rules#rule_key=squid%3AUnusedPrivateMethod)